src/System.Private.DataContractSerialization/src/System.Private.DataContractSerialization.csproj

```diff
@@ -162,10 +162,23 @@
     <Compile Include="System\Xml\IFragmentCapableXmlDictionaryWriter.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)'=='uapaot'">
-    <ReferenceFromRuntime Include="System.Private.CoreLib.Augments" />
+    <ReferenceFromRuntime Include="System.Private.CoreLib" />
```

Comment:
This is not right. Instead, you should:
- Delete reference to `<ReferenceFromRuntime Include="System.Private.CoreLib.Augments" />` from here
- Delete the two `#if NET_NATIVE` ifdefs from System/Runtime/Serialization/XmlFormatReaderGenerator.cs that you will hit the build breaks for. There does not need to be divergence with CoreCLR anymore.